src/components/Button/Button.js

```diff
@@ -0,0 +1,22 @@
+import { Loader } from "../Loader/Loader";
+import { LabelWrapper, StyledButton } from "../Button/Button.style";
+import PropTypes from "prop-types";
+
+export const Button = ({ type, onClick, label, isLoading, ...props }) => {
```

Comment:
we rather want to use button this way:
```
<Button
     onClick={() => null}
     isLoading
>
   Submit
</Button>
```
use children prop then instead of label prop